Running tests with rendering fails

At the moment if we run tests without rendering, it fails:

```python
self = PosixPath('/Users/aktech/.papyri/html/p/numpy/1.23.4/api/numpy.html'), name = '/Users/aktech/.papyri/html/p/numpy/1.23.4/api/numpy.html', flags = 16777216, mode = 438

    def _opener(self, name, flags, mode=0o666):
        # A stub for the opener argument to built-in open()
>       return self._accessor.open(self, flags, mode)
E       FileNotFoundError: [Errno 2] No such file or directory: '/Users/aktech/.papyri/html/p/numpy/1.23.4/api/numpy.html'

../../miniconda3/envs/papyri/lib/python3.9/pathlib.py:1120: FileNotFoundError
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+__pycache__
 .pytest_cache/
 papyri/rst.so
 cache

--- a/papyri/tests/test_full_render.py
+++ b/papyri/tests/test_full_render.py
@@ -16,11 +16,12 @@ PRESENT_PATTERNS = [
 
 @pytest.mark.parametrize("path, reason, expected", PRESENT_PATTERNS)
 def test_render(path, reason, expected):
-    html_path = (html_dir / path)
+    html_path = html_dir / path
     if not html_path.exists():
         logging.info("papyri render not done, rendering now.")
         from papyri.render import main as m2
         import trio
+
         trio.run(m2, False, True, False, True, True, False)
     text = html_path.read_text()
     assert expected in text, reason

--- a/papyri/tests/test_full_render.py
+++ b/papyri/tests/test_full_render.py
@@ -1,6 +1,9 @@
+import logging
+
 from papyri.config import html_dir
 import pytest
 
+log = logging.getLogger("papyri")
 
 PRESENT_PATTERNS = [
     (
@@ -13,5 +16,11 @@ PRESENT_PATTERNS = [
 
 @pytest.mark.parametrize("path, reason, expected", PRESENT_PATTERNS)
 def test_render(path, reason, expected):
-    text = (html_dir / path).read_text()
+    html_path = (html_dir / path)
+    if not html_path.exists():
+        logging.info("papyri render not done, rendering now.")
+        from papyri.render import main as m2
+        import trio
+        trio.run(m2, False, True, False, True, True, False)
+    text = html_path.read_text()
     assert expected in text, reason

